backend/molgenis-emx2-webapi/build.gradle

```diff
@@ -11,7 +11,7 @@ dependencies {
     implementation project(':backend:molgenis-emx2-tasks')
     implementation project(':backend:molgenis-emx2-email')
     implementation project(':backend:molgenis-emx2-analytics')
-    implementation 'org.eclipse.rdf4j:rdf4j-rio-api:4.3.12'
+    implementation 'org.eclipse.rdf4j:rdf4j-rio-api:5.0.1'
```

Comment:
is die dependency echt nodig? Heb je geprobeerd hem te deleten?

Reply:
Niet getest, zal ik doen. Ik vermoed dat dit er staat vanwege: https://github.com/molgenis/molgenis-emx2/blob/c47098a976bc2b1db7115b39c0810baecafa82bc/backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/RDFApi.java#L16

Maar mogelijk dat dependency naar molgenis-emx2-rdf dit ook al dekt.

Reply:
ah nee check, het staat ook al in de parent `build.gradle`. Ik verwijder hem hier.